### DISCUSSION TITLE
Spoolman wrong calculate

### DISCUSSION THREAD

**Discussion by Keksianer** [Donkie/Spoolman#352] [Q&A] [answered]
Hello there.
I build Voron Legacy from my Ender 5 Pro.
Before that, i used Marlin with OctoPrint and Filament Manager, Cura as Slicer, it works fine.
But now i have a Problem: In Spoolman i see i have around 150g Filament on Spool, but in real i got only around 30g.
I have no Problem with it, if Spoolman says i have 100g and i got 130g on Spool.
In that case it is ok for me to calculate my prints, but if i have less more than in Spoolman, it is not really good for me to calc.
I have Ikea Sammla as Dry Box.
My Question: Are there some settings that i have to change? Density values are correct.
I know it must not be Perfect, i do not want know if i have 101g oder 102g, but it should be in a range of 10g for me, if that is possible?
I mean 1kg Spool and 100g less is in my opinion too much.
Hope you can help me.

**Comment by Donkie** [comment, marked as answer]
Please see this https://github.com/Donkie/Spoolman/wiki/Frequently-Asked-Questions#filament-usage-is-very-incorrect if you still have issues.
